lib/Interpreter/CppInterOp.cpp

```diff
@@ -3178,35 +3178,53 @@ namespace Cpp {
 
     clang::UnresolvedSet<8> lookup;
 
-    if (op == "+")
+    switch (op) {
+    case Plus:
       getSema().LookupOverloadedOperatorName(clang::OO_Plus, S, lookup);
```

Comment:
Can't we follow the enum ordering in clang. Eg clang::OO_Plus == BinaryOperator.Plus? Then this will be just a single call instead of a switch stmt.